client/src/templates/Challenges/components/side-panel.tsx

```diff
@@ -69,12 +66,21 @@ export function SidePanel({
     >
       {challengeTitle}
       {hasDemo && (
-        <>
-          <Button size='small' onClick={() => openModal('projectPreview')}>
-            {t('buttons.show-demo')}
-          </Button>
-          <Spacer size='xSmall' />
-        </>
+        <p>
+          <Trans i18nKey='learn.example-app'>
+            <span
+              className='example-app-link'
+              onClick={() => openModal('projectPreview')}
+              role='button'
+              tabIndex={0}
+              onKeyDown={e => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  openModal('projectPreview');
+                }
+              }}
+            ></span>
```

Comment:
Instead of a span, I would have made this a `button` with whatever styling is needed to make it look like the `a` element we want. I suppose with `role` of button, this does not make much of a difference.

Reply:
I debated between the two and used button initially, but decided to go with span. There was a reason, but I don't recall off hand. I think there was some styling that I couldn't override to make it look like the other links.